Packs/ApiModules/Scripts/JSONFeedApiModule/JSONFeedApiModule.py

```diff
@@ -333,6 +335,50 @@ def determine_indicator_type(indicator_type, auto_detect, value):
     return indicator_type
 
 
+def merge_indicators_service(first_indicator, second_indicator):
+    """
+    Gets two indicators with the same value and different services (tags) and merges them.
+    Args:
+        first_indicator: (dict) The first indicator.
+        second_indicator: (dict) The second indicator.
+    Returns:
+        A dict of the merged indicator.
+    """
+    first_indicator['rawJSON']['service'] += f", {second_indicator.get('rawJSON', {}).get('service', '')}"
+    return first_indicator
+
+
+def deduplicate_indicators(indicators):
```

Comment:
Add the type for the function signature and return type to annotation, e.g. 
def deduplicate_indicators(indicators: List) -> List: